R/ChromBackend.R

```diff
@@ -134,28 +134,24 @@
 #'   correctly.
 #'
 #' - `chromData()`, `chromData<-`: gets or sets general chromatogram metadata
-#'   (annotation).  `chromData()` returns a `DataFrame`, `chromData<-` expects
+#'   (annotation). `chromData()` returns a `DataFrame`, `chromData<-` expects
```

Comment:
I would suggest you also describe the `drop` parameter here to clearly define what `drop = TRUE` and `drop = FALSE` is expected to do. Same also for the `peaksData()` documentation below. And you also need to have a `@param drop` documentation entry (maybe you have it already but I did not see it here). We need to be very clear with this parameter, especially because we don't have that in *Spectra*.